main.go

```diff
@@ -13,11 +13,13 @@ import (
 
 func main() {
 	target := flag.String("target", "depesz", "type of an explain visualizer to export")
+	url := flag.String("post_url", "", "Absolute URL to HTML <form> element's `action`")
+	key := flag.String("plan_key", "", "`name` of HTML <textarea> element where query plan gets pasted")
 
 	flag.Parse()
 
 	ctx := context.Background()
-	planner, err := visualizer.New(*target)
+	planner, err := visualizer.New(*target, *url, *key)
```

Comment:
We could use a separate `Config` struct to pass parameters since there can be quite a lot of them.

Reply:
👍

Reply:
I haven't implemented this yet -- we are now at two args in the `New()` instantiator.  I can create the `Config` struct, but I wanted to confirm with you that you think this is the way to go.  Basically, 1) create a new package and import to `main.go` and `visualizer.go`, or 2) just pass two args?

Reply:
I think the first option is better: create a separate package for the `Config` structure and then pass it to the visualizer. So, an increase in the number of options will not lead to a change in the high-level API.